### PR TITLE
add description column to table in README

### PR DESCRIPTION
**Pull request by Loquacity**
> __ALL Pull-Requests require an associated [Issue](https://github.com/thegooddocsproject/templates/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc) as documented in [How to contribute](https://github.com/thegooddocsproject/templates/blob/master/contribute.md#contributing).__

## Purpose / why

closes: https://github.com/thegooddocsproject/templates/issues/77

## What changes were made?

We added a column to the `readme` with template information

## Verification

N/A

---

## Checklist

> __Pull-request reviewer__ should ensure the following

* [x] Are issues linked correctly?
* [x] Is this PR labeled correctly?
* [x] If template updates: do they align with [developers.google.com/style/](https://developers.google.com/style/)?
* [x] Did the PR receive at least one :+1: and no :-1: from core-maintainers?
* [x] On merging, did you complete the merge using [keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#manually-linking-a-pull-request-to-an-issue)?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,18 +33,18 @@ When writing your documentation, it helps to think about the following:
...
-Discussion | Concept
-How-to | Task
-Tutorial | Concept, Task
-General reference entry | Reference
-Logging reference | Reference
+Current templates:
+
+| Template name | Documentation type | Description |
+| ------------- | ------------------ | ----------- |
+| API Project overview | Concept | An overview of your API
```
Comment (waldyrious) [inline README.md:40]:
```suggestion
| API Project overview | Concept | An overview of your API |
```

File: README.md [hunk 2]
```diff
@@ -33,18 +33,18 @@ When writing your documentation, it helps to think about the following:
...
+| API Quickstart | Concept, Task | Simplest possible method of implementing your API |
+| API Reference | Reference | List of references related to your API |
+| Discussion | Concept | Longer document giving background or context to a topic |
+| How-to | Task | Short series of steps for a particular task
```
Comment (waldyrious) [inline README.md:44]:
```suggestion
| How-to | Task | Short series of steps for a particular task |
```

File: README.md [hunk 3]
```diff
@@ -33,18 +33,18 @@ When writing your documentation, it helps to think about the following:
...
+| Tutorial | Concept, Task | An training document for a product or topic |
```
Comment (waldyrious) [inline README.md:45]:
```suggestion
| Tutorial | Concept, Task | A training document for a product or topic |
```

File: README.md [hunk 4]
```diff
@@ -33,18 +33,18 @@ When writing your documentation, it helps to think about the following:
...
+| General reference entry | Reference | Specific details about a particular topic |
+| Logging reference | Reference | Describes log pipelines |
```
Comment (waldyrious) [inline README.md:47]:
```suggestion
| Logging reference | Reference | Description of log pipelines |
```

**Comment by mgan59** [comment]
Reviewed during the 08/26 meeting.  Everyone on the call agree this look good to merge 👍

**Pull request merged**